tests/memory_tests/test_runner/conftest.py

```diff
@@ -0,0 +1,547 @@
+        help='Directory to put test data into. Required for OMZ converter.py only.'
+    )
+    helpers_args_parser = parser.getgroup("test helpers")
+    helpers_args_parser.addoption(
+        "--dump_refs",
+        type=str,
+        help="path to dump test config with references updated with statistics collected while run",
+    )
+    db_args_parser = parser.getgroup("test database use")
+    db_args_parser.addoption(
+        '--db_submit',
+        metavar="RUN_ID",
+        type=str,
+        help='submit results to the database. ' \
+             '`RUN_ID` should be a string uniquely identifying the run' \
+             ' (like Jenkins URL or time)'
+    )
+    is_db_used = db_args_parser.parser.parse_known_args(sys.argv).db_submit
+    db_args_parser.addoption(
+        '--db_url',
+        type=str,
+        required=is_db_used,
+        help='MongoDB URL in a form "mongodb://server:port"'
+    )
+    db_args_parser.addoption(
+        '--timeline_report',
+        type=str,
+        required=is_db_used,
+        help='path to build manifest to extract commit information'
+    )
+    db_args_parser.addoption(
+        '--db_name',
+        type=str,
+        required=is_db_used,
+        help='database name',
+        choices=DATABASES
+    )
+    db_args_parser.addoption(
+        '--db_collection',
+        type=str,
+        required=is_db_used,
+        help='collection name in database',
+        choices=DB_COLLECTIONS
+    )
+    db_args_parser.addoption(
+        '--db_metadata',
+        type=str,
+        default=None,
+        help='path to JSON-formatted file to extract additional information'
+    )
+    db_args_parser.addoption(
+        '--manifest',
+        type=Path,
+        required=is_db_used,
+        help='path to build manifest to extract commit information'
+    )
+
+
+@pytest.fixture(scope="session")
+def executable(request):
+    """Fixture function for command-line option."""
+    return request.config.getoption('executable')
+
+
+@pytest.fixture(scope="session")
+def niter(request):
+    """Fixture function for command-line option."""
+    return request.config.getoption('niter')
+
+
+# -------------------- CLI options --------------------
+
+
+@pytest.fixture(scope="function")
+def omz_models_conversion(pytestconfig, request):
+    """
+    Fixture for preparing omz models and updating test config with new paths
+    """
+    omz_path = request.config.getoption("omz_repo")
+    cache_dir = request.config.getoption("omz_cache_dir")
+    omz_models_out_dir = request.config.getoption("omz_models_out_dir")
+    omz_irs_out_dir = request.config.getoption("omz_irs_out_dir")
+    mo_path = request.config.getoption("mo_path")
+
+    downloader_path = omz_path / "tools" / "downloader" / "downloader.py"
+    converter_path = omz_path / "tools" / "downloader" / "converter.py"
+    info_dumper_path = omz_path / "tools" / "downloader" / "info_dumper.py"
+
+    records = [rec for rec in pytestconfig.session_info]
```

Comment:
since fixture has scope=function, fixture status will be a part of test status. In this case fixture should prepare only the model from specific test, not for all run

I suggest doing conversion model per test in order to spread conversion status to the test status - it will help to maintain issues related to OMZ or MO easier